pkgs/development/python-modules/ytmusicapi/default.nix

```diff
@@ -37,7 +37,6 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python API for YouTube Music";
     homepage = "https://github.com/sigma67/ytmusicapi";
-    changelog = "https://github.com/sigma67/ytmusicapi/releases/tag/${version}";
```

Comment:
It actually does provide a changelog in this case.

Reply:
Please verify it is the correct URL now